File: app/src/main/java/com/astinil/AndroidTimesheet/api/model/TimeSheetRequest.java
package com.astinil.AndroidTimesheet.api.model;

public class TimeSheetRequest {
    public String date;
    public String attendanceType;  // CHECK_IN or CHECK_OUT
    public String remarks;

    public TimeSheetRequest() {}

    public TimeSheetRequest(String date, String attendanceType, String remarks) {
        this.date = date;
        this.attendanceType = attendanceType;
        this.remarks = remarks;
    }
}

File: app/src/main/java/com/astinil/AndroidTimesheet/api/model/TimeSheetRequest.kt
package com.astinil.AndroidTimesheet.api.model


import com.astinil.AndroidTimesheet.api.model.TimesheetEntryRequest;
data class TimeSheetRequest(
    val timesheetType: String,
    val startDate: String,
    val endDate: String,
    val job: String,
    val projectId: String,
    val entries: List<TimesheetEntryRequest>
)
